prow/plugins/golint/suggestion/golint_suggestion.go

```diff
@@ -76,6 +78,18 @@ func fixNameAllCaps(p lint.Problem) string {
 	return result
 }
 
+func fixStutter(p lint.Problem) string {
+	matches := lintStutter.FindStringSubmatch(p.Text)
+	if len(matches) < 3 {
+		return ""
+	}
+	suggestion := strings.Replace(p.LineText, matches[1][(strings.Index(matches[1], ".") + 1) :], matches[2], -1)
```

Comment:
Instead of doing extra string processing here you should just change the first matching group of the regexp to only match the part that you want.
So instead of `(.*)` use `[^.]+\.(.*)`